src/lib/error-rate-check.js

```diff
@@ -0,0 +1,23 @@
+const nHealth = require('n-health');
+
+module.exports = (appName) => (
+	nHealth.runCheck({
+		type: 'graphiteThreshold',
+		metric: `
+			divideSeries(
+				sumSeries(
+					next.heroku.${appName}.web_*_${process.env.REGION || '*'}.express.default_route_GET.res.status.500.count
+				),
+				sumSeries(
+					next.heroku.${appName}.web_*_${process.env.REGION || '*'}.express.default_route_GET.res.status.*.count
+				)
+			)
+		`.replace(/\t|\n/g, ''),
+		threshold: 0.02,
+		name: `500 rate for ${appName} is acceptable`,
+		severity: 2,
```

Comment:
I would default this to 3 (or 1 - I can't remember which is the highest level), but possibly allow configuring it as an n-express option - some apps (particularly experimental ones) may error a lot